Bug: https://bugs.debian.org/1041124
From 3f8bc3343983697531d52918fb6cdba19593f3a8 Mon Sep 17 00:00:00 2001
From: Gianfranco Costamagna <costamagnagianfranco@yahoo.it>
Date: Sat, 15 Jul 2023 11:31:55 +0200
Subject: [PATCH] Update CMakeLists.txt: set required std version to gnu++14

New googletest 1.13.0 defaults to gnu++14, so makes this package FTBFS when building testsuite with this error:

In file included from /usr/include/gtest/gtest-message.h:57,
                 from /usr/include/gtest/gtest-assertion-result.h:46,
                 from /usr/include/gtest/gtest.h:64,
                 from /build/1st/dlt-daemon-2.18.9/tests/gtest_dlt_user.cpp:27:
/usr/include/gtest/internal/gtest-port.h:270:2: error: #error C++ versions less than C++14 are not supported.
  270 | #error C++ versions less than C++14 are not supported.
      |  ^~~~~

Thanks Adrian Bunk <bunk@debian.org> for the report and the fix!
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 014fc3bb..b2e056db 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -213,7 +213,7 @@ endif()
 
 add_compile_options(
     $<$<COMPILE_LANGUAGE:C>:-std=gnu99>
-    $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++11>
+    $<$<COMPILE_LANGUAGE:CXX>:-std=gnu++14>
     -Wall
     -Wextra
     # -pedantic
